src/Goerli-DssSpell.sol

```diff
@@ -40,6 +34,14 @@ contract DssSpellAction is DssAction {
         return false;
     }
 
+    address constant internal MCD_CLIP_CALC_GUSD_A = 0x738EA932C2aFb1D8e47bebB7ed1c604399f2A99e;
+    address constant internal MCD_CLIP_CALC_USDC_A = 0x3a278aA4264AD66c5DEaAfbC1fCf6E43ceD47325;
+    address constant internal MCD_CLIP_CALC_PAXUSD_A = 0x8EE38002052CA938646F653831E9a6Af6Cc8BeBf;
+
+    address immutable internal STARKNET_GOV_RELAY = DssExecLib.getChangelogAddress("STARKNET_GOV_RELAY");
+    address constant internal NEW_STARKNET_GOV_RELAY = 0x8919aefA417745F22c6af5AD6550E83159a373F3;
+    uint256 constant internal L2_GOV_RELAY_SPELL = 0x04c93f9818a4f81f6f2c6f0f660cb4986b789b6b6fb1b274b879649deed74eb8;
```

Comment:
just run `amarna l2spell.cairo -s` (amarna `v0.1.5`):
- [unused-imports] in l2spell.cairo:4:48

Reply:
Turns out our amarna CI config was not correct. Fixed nod.

Reply:
L2 mainnet spell will be redeployed?